### DISCUSSION TITLE
Any way to use m2m100?

### DISCUSSION THREAD

**Discussion by IcyAlmond** [dmMaze/BallonsTranslator#211] [Q&A] [answered]
https://github.com/zyddnys/manga-image-translator supports m2m100 bu I can't seem to find it in this program?

![image](https://github.com/dmMaze/BallonsTranslator/assets/67372293/3db36519-dd38-497e-95b7-15eb379cb048)

**Comment by bropines** [comment, marked as answer]
> supports m2m100 bu I can't seem to find it in this program?

DmMaze and we are working slowly, but I think we will add it in the future

**Comment by bropines** [comment]
> https://github.com/zyddnys/manga-image-translator supports m2m100 bu I can't seem to find it in this program?
> 
> ![image](https://private-user-images.githubusercontent.com/67372293/265689407-3db36519-dd38-497e-95b7-15eb379cb048.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDQ5NDEzNTYsIm5iZiI6MTcwNDk0MTA1NiwicGF0aCI6Ii82NzM3MjI5My8yNjU2ODk0MDctM2RiMzY1MTktZGQzOC00OTdlLTk1YjctMTVlYjM3OWNiMDQ4LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAxMTElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMTExVDAyNDQxNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTJiNDVhZmMwZTFmYzVjZjc1YjY5ZTI2NmQyMzgxZjM5OWI3OWExNmM5Nzc0OTdiZWI1NzM1NDdjNWYxMGJmNWQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.HBi5nhPMWr6hGItxtNsMLrbwq4-EqyKtOuhIfRn4p-U)

I plan to look into this, but I can't find a repository with an implementation of this translator, for reference. Can you help?
